packages/ngworker/lumberjack/http-driver/src/lib/configuration/lumberjack-http-driver.module.ts

```diff
@@ -27,7 +29,12 @@ export class LumberjackHttpDriverModule {
       providers: [
         {
           provide: lumberjackHttpDriverConfigToken,
-          useValue: config,
+          deps: [lumberjackLogDriverConfigToken],
+          useFactory: (logDriverConfig: LumberjackLogDriverConfig) => ({
```

Comment:
suggestion: add explicit return type to prevent type errors.
```suggestion
          useFactory: (logDriverConfig: LumberjackLogDriverConfig): LumberjackHttpDriverInternalConfig => ({
```